### PR TITLE
iCal feeds config, parser, and item

### PR DESCRIPTION
**Pull request by jordan-caldwell**
## Summary
This PR adds a custom iCal parser included in a custom module ecms_feeds. This also includes the necessary config to add the feed type to existing ecms_event feature package.

## Metadata

| Question | Answer |
|----------|--------|
| Did you use a meaningful pull request title? | yes
| Did you apply meaningful labels to the pull request? | yes
| Did you perform a self review first? | yes
| `CHANGELOG` reflects changes? | yes
| Did you perform browser testing? | no
| Risk level | Low
| Relevant links | https://thinkoomph.jira.com/browse/RIG-251

### CODE REVIEW

**Comment by jordan-caldwell** [comment]
@ben-hamelin This is out of WIP and ready for review.

File: ecms_base/features/custom/ecms_event/config/install/field.field.node.event.feeds_item.yml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  module:
+    - feeds
+id: node.event.feeds_item
+field_name: feeds_item
+entity_type: node
+bundle: event
+label: 'Feeds item'
+description: ''
+required: false
+translatable: true
```
Comment (ben-hamelin) [inline ecms_base/features/custom/ecms_event/config/install/field.field.node.event.feeds_item.yml:16]: should this be translatable false?

File: ecms_base/modules/custom/ecms_feeds/src/Feeds/Item/EcmsIcalItem.php [hunk 2]
```diff
@@ -0,0 +1,73 @@
+<?php
+
```
Comment (ben-hamelin) [inline ecms_base/modules/custom/ecms_feeds/src/Feeds/Item/EcmsIcalItem.php:2]: add `declare(strict_types = 1);`

**Review by ben-hamelin — CHANGES_REQUESTED** [review]
just one minor addition

**Comment by ben-hamelin** [comment]
Ok!
/approved

**Pull request merged**